sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -8831,6 +8831,23 @@ pi_result _pi_buffer::free() {
 
 /// command-buffer Extension
 
+/// Helper function to take a list of pi_ext_sync_points and fill the provided
+/// vector with the associated ZeEvents
+pi_result getEventsFromSyncPoints(
```

Comment:
can we use `static` or an anonymous namespace to show that this function won't be used elsewhere.